l10n_uy_ux/models/account_move.py

```diff
@@ -52,8 +53,64 @@ def _l10n_uy_edi_check_move(self):
 
         return errors
 
+    def _post(self, soft=True):
+        # EXTENDS l10n_uy_edi
+        res = super()._post(soft=soft)
+        if electronic_moves := self.filtered(lambda m: m.journal_id.l10n_uy_edi_type == "electronic"):
+            if len(electronic_moves) == 1:
+                # Para una sola factura usar el wizard individual
+                wizard_vals = {
+                    "move_id": electronic_moves.id,
+                }
+                # Si la factura usa plantilla de mail, asignarla
+                if electronic_moves.journal_id.mail_template_id:
+                    wizard_vals["mail_template_id"] = electronic_moves.journal_id.mail_template_id.id
+                else:
+                    wizard_vals["sending_method_checkboxes"] = {"email": {"checked": False}}
+                    wizard_vals["sending_methods"] = []
+                wizard = self.env["account.move.send.wizard"].sudo().create(wizard_vals)
+                wizard.action_send_and_print()
+            else:
+                # Para múltiples facturas usar el wizard batch
+                wizard_vals = {
+                    "move_ids": [(6, 0, electronic_moves.ids)],
+                }
+                # Si todas las facturas usan la misma plantilla de mail, asignarlas. Si no, dejar vacío
+                # TODO: revisar comportamiento de envio de mails en batch
+                wizard = self.env["account.move.send.batch.wizard"].sudo().create(wizard_vals)
+                mail_templates = electronic_moves.mapped("journal_id.mail_template_id")
+                if not len(mail_templates) == 1:
+                    for move in electronic_moves:
+                        sending_settings = wizard._get_default_sending_settings(move)
+                        sending_settings["mail_template"] = None
+                        sending_settings["sending_methods"] = []
+
+                wizard.action_send_and_print(force_synchronous=True)
+        return res
```

Comment:
intentaria no hacer el envio de emils en el post, que el post se encargue solo del post y crear en dgi, y que el luego el proceso del envio de correo siga con el proceso automatico del account_ux

usaria el mismo campo que se usa en el send and print para saber si debe estar activo o no el checkbox Crear CFE (move.l10n_uy_edi_is_needed)

```suggestion
            for move in self.filtered(lambda m: m.journal_id.l10n_uy_edi_is_needed):
                move._l10n_uy_edi_send()
                if move.l10n_uy_edi_error:
                    message.post("error al intentar validar en DGI %s" % move.l10n_uy_edi_error)
                    move.button_draft()                    
        return res
```